### PR TITLE
Survey functionality  Indkom's business pages

### PR DESCRIPTION
**Pull request by bendikstoasannes**

### CODE REVIEW

**Comment by yoonsiko** [comment]
I can't seem to edit the answers to the questions. This is crucial since when creating a new question from admin, there seems to be no problems, but when a member from Industrikomiteen are creating a new question a Placeholder question is created, which can't be edited. At least from my local. Tried to look at the answer_edit function in views.py, but I couldn't locate any errors. Are you able to edit them?

File: chemie/corporate/views.py [hunk 1]
```diff
@@ -130,3 +168,197 @@ def job_remove(request, id):
...
+def answer_edit(request):
+    if request.method == "POST":
+        id = request.POST["id"]
+        value = request.POST["value"]
+        answer = request.POST["answer"]
+
+        answer_value_pair = get_object_or_404(AnswerKeyValuePair, id=id)
+        answer_value_pair.value = value
+        answer_value_pair.answer = answer
+
```
Comment (yoonsiko) [inline chemie/corporate/views.py:347]: Change value and answer to edited_value and edited_answer, respectively (I think)

**Comment by yoonsiko** [comment]
Editing the question seems fine, but it's not intuitive to find the edit button (rediger) to edit the question. Took me couple of minutes to find out how to change the questions, but it really should be in the first admin panel

File: chemie/corporate/views.py [hunk 2]
```diff
@@ -130,3 +168,197 @@ def job_remove(request, id):
...
+@permission_required("corporate:change_answerkeyvaluepair")
+def answer_edit(request):
+    if request.method == "POST":
+        id = request.POST["id"]
+        value = request.POST["value"]
+        answer = request.POST["answer"]
+
+        answer_value_pair = get_object_or_404(AnswerKeyValuePair, id=id)
+        answer_value_pair.value = value
+        answer_value_pair.answer = answer
```
Comment (yoonsiko) [inline chemie/corporate/views.py:346]: Change "answer_value_pair.**answer**" to "answer_value_pair.**key**", as the AnswerKeyValuePair model has the variable name "key" as it's "answer".

**Comment by yoonsiko** [comment]
When deleting the answers to the question, it removes the question to the survey. Probably a button with wrong function, where it deleted the question, not the answer.

**Comment by yoonsiko** [comment]
A suggestion to a improvement. It could be nice to see the total answers and a percentage of each answer.

File: chemie/corporate/views.py (hunk 2 above)
Comment (bendikstoasannes) [reply chemie/corporate/views.py:346]: You are absolutely right. Good catch! It is corrected now

**Review by yoonsiko — APPROVED** [review]
Seems like everything works!

**Pull request merged**